hail/src/main/scala/is/hail/asm4s/Code.scala

```diff
@@ -18,6 +18,13 @@ object Code {
     }
   }
 
+  def apply[S1](c1: Code[S1]): Code[S1] =
```

Comment:
This function should just return `c1` unchanged, but I think I might prefer if the typing issue was addressed in either of the other ways I mentioned. (up to you.)

Reply:
Yeah, I'll just get rid of this and use const.